test/core/error_tracker.test.ts

```diff
@@ -1,7 +1,5 @@
 /* eslint-disable no-multi-spaces */
```

Comment:
Can you add or update unit test that will look into the tracker object if there correct links are stored?

Reply:
I already added them in e2e tests as we need the context, and I just made a couple of updates to them in the latest commit